src/app/App.js

```diff
@@ -943,7 +943,18 @@ class App extends EventEmitter {
 			console.log('Navigate aborted, invalid mouse button or modifier key pressed.');
 			return;
 		}
-		this.maybeNavigate_(event.delegateTarget.href, event);
+
+		let href = event.delegateTarget.href;
+
+		// If SVGAnimatedString object is passed as url
+		if (isObject(href) && href.toString() === '[object SVGAnimatedString]') {
```

Comment:
Hey @diegonvs, this looks overly specific and way too complicated... isn't there a better way to deal with this? Can't we query the svg dom in a more meaningful way?